machines/bleShare/scan/scanMachine.ts

```diff
@@ -137,6 +139,26 @@ export const scanMachine =
         inactive: {
           entry: 'removeLoggers',
         },
+        checkStorage: {
+          invoke: {
+            src: () =>
+              Promise.resolve(
+                Storage.isMinimumLimitReached(
+                  'minimumStorageRequiredForAuditEntryInMB'
```

Comment:
do we need to mention inMB here? we have comments added for every property in the mosip-config. we can do the same for this property as well.

Reply:
Refactored